acid/utils.py

```diff
@@ -7,3 +8,10 @@ def pipe_intersect(pipelines_config, pipelines_json):
 
     return [pipeline['name'] for pipeline in pipelines_json
             if pipeline['name'] in pipelines_config]
+
+
+def get_feature_logger(feature_name=None):
```

Comment:
please change name for something more universal like: app_loger() etc.

Reply:
`get_feature_logger` -> `get_logger`